src/gtk/dnd.cpp

```diff
@@ -919,6 +919,12 @@ wxDragResult wxDropSource::DoDragDrop(int flags)
 
     g_signal_handlers_disconnect_by_func (m_iconWindow,
                                           (gpointer) gtk_dnd_window_configure_callback, this);
+
+#ifdef __WXGTK3__
+    g_signal_handlers_disconnect_matched (m_iconWindow,
```

Comment:
This is minor, but why can't we use `g_signal_handlers_disconnect_by_func` here for consistency?

Reply:
We can't pass data (the latest argument) (It's not a member of class),
So I've decided to use `g_signal_handlers_disconnect_matched` instead.